protobackend/output.py

```diff
@@ -55,10 +59,15 @@ def wrapper(*args, **kwargs):
         with CaptureErrors(fallback_output):
             output = f(*args, **kwargs)
             if not isinstance(output, list):
+                if output is not None:
```

Comment:
If I get it right, `None` returns an empty array but that's a wanted behavior? I suggest either to log in that case or mention is on the description.

Reply:
`None` is the implicit return value, which allows commands that just do something without output to not have `return []`.